spec/mcode.spec.ts

```diff
@@ -874,7 +894,7 @@ describe('checkSurgicalProcedureFilterLogic-Splenectomy', () => {
   });
 });
 
-/* Medication Statement Logic Tests */
+/* Medication Statement Logic Tests */ // - needs updated
```

Comment:
Same question as above - does this still need updating?

Reply:
Updates have been made. Comment removed.